scripts/openstack-quickstart-demosetup

```diff
@@ -466,7 +470,7 @@ done
 ## setup loopback LVM configuration
 #-----------------------------------------
 
-
+start_and_enable_service lvm2-lvmetad
```

Comment:
Hm. I like the current solution more. Looks simpler to me. But I could also add ```systemctl enable lvm2-lvmetad; systemctl start lvm2-lvmetad``` to the loopback lvm script. then we don't need the inclusion of functions.sh.